.github/workflows/hz-dockerhub-release.yml

```
@@ -0,0 +1,307 @@
+          cp -r ./operator-repo/${NAME}-operator ./${NAME}-op
+
+          sed -i -e  "s/imagePullPolicy: .*/imagePullPolicy: Never/g" ./${NAME}-op/operator.yaml
+          sed -i -e  "s/productVersion: .*/productVersion: ${OPERATOR_VERSION}/g" ./${NAME}-op/operator.yaml
+          sed -i -e  "s/image: hazelcast\/${OPERATOR_NAME}:.*/image: hazelcast\/${OPERATOR_NAME}:${OPERATOR_VERSION}/g" ./${NAME}-op/operator.yaml
+
+          sed -i -e "0,/tag: .*/s//tag: ${DEFAULT_HAZELCAST_VERSION}/" ./${NAME}-op/hazelcast.yaml
+          sed -i -e "0,/tag:/! s/tag: .*/tag: ${DEFAULT_MANCENTER_VERSION}/" ./${NAME}-op/hazelcast.yaml
+
+
+
+      - name: Upload Hazelcast-op files
+        uses: actions/upload-artifact@v2
+        with:
+          name: hazelcast-operator
+          path: ./hazelcast-op 
+
+
+      - name: Set up Kubernetes in Docker
+        uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.7.0"
+
+      - name: Deploy Hazelcast Cluster and Validate the Cluster
+        run: |
+          cd ./${NAME}-op
+          
+          OPERATOR_NAME="${NAME}-operator"
+          OPERATOR_REPOSITORY="hazelcast/${OPERATOR_NAME}"
+          OPERATOR_IMAGE="${OPERATOR_REPOSITORY}:${OPERATOR_VERSION}" 
+
+          kind load docker-image ${OPERATOR_IMAGE}
+
+          kubectl apply -f operator-rbac.yaml
+          kubectl apply -f hazelcast-rbac.yaml
+          kubectl apply -f hazelcastcluster.crd.yaml
+          kubectl --validate=false apply -f operator.yaml
+
+          #WAIT FOR OPERATOR TO BE READY
+          for i in `seq 1 15`; do
+            if [[ $(kubectl get pods -l app.kubernetes.io/name=${OPERATOR_NAME} -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; then
+              echo "waiting for Hazelcast operator pod to be ready, " && sleep 10
+              if [ "$i" = "15" ]; then
+                echo "Hazelcast operator pod failed to be ready!"
+                kubectl get pods
+                echo ""
+                kubectl logs -l app.kubernetes.io/name=${OPERATOR_NAME}
+                exit 1
+              fi
+            else
+              echo "Hazelcast operator pod is ready!"
+              break
+            fi
+          done
+
+          kubectl apply -f hazelcast.yaml
+          kubectl logs -l app.kubernetes.io/name=${OPERATOR_NAME}
+
+
+          #WAIT FOR hz-hazelcast-2 TO BE READY
+          for i in `seq 1 15`; do
+            if [[ $(kubectl get pods hz-${NAME}-2 -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; then
+              echo "waiting for hz-${NAME}-2 pod" && sleep 10
+              if [ "$i" = "15" ]; then
+                echo "hz-${NAME}-2 pod failed to be ready!"
+                kubectl get pods
+                echo ""
+                kubectl logs hz-${NAME}-2
+                exit 1
+              fi
+            else
+              echo "hz-${NAME}-2 is ready!"
+              break
+            fi
+          done
```

```suggestion
 kubectl wait --for=condition=Ready --timeout=60s pod -l _LABEL_OF_HZ_POD_=${OPERATOR_NAME}
```
You can use one of below labels as a `_LABEL_OF_HZ_POD_`:
https://github.com/hazelcast/hazelcast-operator/blob/master/hazelcast-operator/hazelcast.yaml#L6-L8